Elsewhen/Views/AltIconView.swift

```diff
@@ -9,32 +9,42 @@ import SwiftUI
 
 struct AltIconView: View {
     
-    #if !os(macOS)
+#if !os(macOS)
     @EnvironmentObject internal var orientationObserver: OrientationObserver
     @Environment(\.horizontalSizeClass) var horizontalSizeClass
-    #endif
+#endif
+    
+    @State private var containerWidth: CGFloat?
     
     @State var viewId = 1
     var done: () -> ()
     
+    var isPadAndNotCompact: Bool {
+        DeviceType.isPad() && horizontalSizeClass != .compact
+    }
+    
+    var columns: Int {
+        Int(containerWidth ?? 300) / 90 - 1
```

Comment:
300 and 90 kinda seem like magic numbers?

Reply:
300 is a random "it'll never be nil because it gets set by the view as soon as it appears, but it's roughly a decent enough break point to make a nice number of icons" and 90 is "the width of one alternative icon plus padding". So they're not "magic" but they do look like it